src/components/buttons/LightPopButton.js

```diff
@@ -12,10 +13,23 @@ const PinkButton = styled.button`
   background: ${c.SECONDARY_POP_COLOR};
   border-radius: 3px;
   border: none;
-
   &:hover {
     cursor: pointer;
   }
+  align-content: center;
+
+  @media (min-width: ${g.phoneMediaBreak}px) {
+    width: 160%;
+    margin-left: -30%;
+    height: 40px;
+    @media (min-width: ${g.desktopMediaBreak}px) {
+      display: flex;
+      flex-direction: column;
+      width: 100%;
+      margin-left: 3%;
+      align-items: center;
+    }
+  }
```

Comment:
Should the media breaks be nested inside each other?

It's also fine since it works but maybe the reason we need to go beyond 100% for this button's width is because the FormContainer has a width of less than 100?

Reply:
Thanks so much Maaruf! That's really great.
Will look into this 👍